### PR TITLE
Correction of typos in the french translation

### PR DESCRIPTION
**Pull request by aidanjungo**
Not found by me, obviously!

### CODE REVIEW

File: frontend/public/locales/fr/translation.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "addOptionalCriterias": "Ajouter les critères optionnels",
     "changeOneVideo": "Changez l'une des deux vidéos pour faire une nouvelle comparaison.",
     "editComparison": "Modifier la comparaison",
-    "comparisonInPublicDataset": "Cette compairaison fait parti des données publiques.",
+    "comparisonInPublicDataset": "Cette comparaison fait parti des données publiques.",
```
Comment (GresilleSiffle) [inline frontend/public/locales/fr/translation.json:18]:
```suggestion
    "comparisonInPublicDataset": "Cette comparaison fait partie des données publiques.",
```

Comment (amatissart) [inline frontend/public/locales/fr/translation.json:18]: To be precise...
```suggestion
    "comparisonInPublicDataset": "Cette comparaison fait partie des données publiques.",
```

Comment (amatissart) [reply frontend/public/locales/fr/translation.json:18]: Argh too late ;) 
cc @GresilleSiffle

**Review by GresilleSiffle — COMMENTED** [review]
Thanks @aidanjungo  :+1: 

I'll review the rest tomorrow.

**Comment by aidanjungo** [comment]
Perfect, you can merge it directly when it's done, if you want.

**Review by lfaucon — APPROVED** [review]

**Pull request merged**